### PR TITLE
Enable static code analysis (tslint) on Travis.

### PR DESCRIPTION
**Pull request by gijskant**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -108,9 +107,6 @@
     "use-life-cycle-interface": true,
     "use-pipe-transform-interface": true,
     "component-class-suffix": true,
-    "directive-class-suffix": true,
-    "no-access-missing-member": true,
-    "templates-use-public": true,
-    "invoke-injectable": true
```
Comment (beaugogh) [inline tslint.json]: why disable these changes?

File: build.gradle [hunk 2]
```diff
@@ -54,6 +54,13 @@ task yarnPackage(type: YarnTask, dependsOn: 'yarn') {
 }
 distTar.dependsOn(yarnPackage)
 
+task yarnLint(type: YarnTask, dependsOn: 'yarn') {
```
Comment (beaugogh) [inline build.gradle]: is this task run somewhere? .travis.yam?

File: tslint.json (hunk 1 above)
Comment (gijskant) [reply tslint.json]: Because they are not supported by tslint, causing warnings.

File: build.gradle (hunk 2 above)
Comment (gijskant) [reply build.gradle]: Check now depends on yarnLint. So, Travis will run those indeed.

**Pull request merged**